runtime/vm/OutOfLineINL_jdk_experimental_value_ValueType.cpp

```diff
@@ -33,14 +33,7 @@ VM_BytecodeAction
 OutOfLineINL_jdk_experimental_value_ValueType_valueClassImpl(J9VMThread *currentThread, J9Method *method)
```

Comment:
There is still a reference to J9VM_OPT_VALHALLA_MVT a few lines above.

Reply:
Yes, I think I've left about four MVT sections in bindnatv.cpp, MHInterpreter.cpp,
OutOfLineINL_jdk_experimental_value_ValueType.cpp, OutOfLineINL.hpp for now, to be edited or removed later. I wasn't too sure about them for now.

Reply:
There will be another PR to remove the rest of the MVT changes